pkg/log/logger.go

```diff
@@ -51,12 +52,23 @@ func NewLogger(level LogLevel) logr.Logger {
 	return NewLoggerTo(os.Stderr, level)
 }
 
+// empty outputPath means no rotation
+// default values: maxSize=100 ie 100 megabytes maxBackups=1000 maxAge=30 ie 30 days
```

Comment:
I'd remove those comments. This file should not know anything about CLI arguments. If we change CLI args there is a high change we forgot to change it here and this comments gets invalid

Reply:
so basically when we do not pass these flags from cli, these values gets applied by default.